controllers/contact.php

```diff
@@ -87,9 +87,13 @@ public function index()
         $fieldMessage->setRequired();
         $form->addElement($fieldMessage);
 
-        $fieldCaptcha = new CaptchaField('captcha');
-        $fieldCaptcha->setLabel($this->text('contactus', 'form_label_captcha'));
-        $form->addElement($fieldCaptcha);
+        if ( ow::getUser()->isAuthenticated() )
```

Comment:
Thanks I can do that. (Not sure what the difference is, I actually copied the condtion blindly from https://github.com/ecki/contactus/blob/topic-conditionalcaptcha/controllers/contact.php#L73 - want me to fix the casing in this place as well? )